packages/jsii-pacmak/test/expected.jsii-calc/dotnet/Amazon.JSII.Tests.CalculatorPackageId/Amazon/JSII/Tests/CalculatorNamespace/Calculator.cs

```diff
@@ -28,7 +28,7 @@ namespace Amazon.JSII.Tests.CalculatorNamespace
     /// Console.WriteLine(calculator.Expression.Value);</code>
     /// </example>
     [JsiiClass(nativeType: typeof(Amazon.JSII.Tests.CalculatorNamespace.Calculator), fullyQualifiedName: "jsii-calc.Calculator", parametersJson: "[{\"docs\":{\"summary\":\"Initialization properties.\"},\"name\":\"props\",\"optional\":true,\"type\":{\"fqn\":\"jsii-calc.CalculatorProps\"}}]")]
-    public class Calculator : Amazon.JSII.Tests.CalculatorNamespace.composition.CompositeOperation
+    public class Calculator : Amazon.JSII.Tests.CalculatorNamespace.Composition.CompositeOperation
```

Comment:
This technically indicates that we have a breaking change here. I am okay with it since we don't use it in the CDK, but I am wondering what's the right thing to do here..

Reply:
Not doing it means permanently adopting non-idiomatic names for namespaces in the various front-end languages.

Reply:
I am okay with hoping for the best.